docs/en/concepts-and-designs/oal.md

```diff
@@ -53,6 +53,12 @@ In this case, input are request of each ServiceInstanceJVMCPU scope, avg is base
 > endpoint_percent = from(Endpoint.*).percent(status == true);
 
 In this case, all input are requests of each endpoint, condition is `endpoint.status == true`.
+- `rate`. The rate expressed as a fraction of 100, for the condition matched input.
```

Comment:
I think you missed `Filter` about the `in`

Reply:
It has been merged into the master branch https://github.com/apache/skywalking/blob/master/docs/en/concepts-and-designs/oal.md#filter